.claude/CLAUDE.md

```
@@ -1,404 +1,111 @@
-- Flag any dependency whose resolved hash doesn't match the lockfile.
-- Never run `npm install` as a troubleshooting step without frozen mode.
-
-### Attribution and Traceability
-
-#### 1.12 AI Contribution Disclosure
-This project does not use Undercover Mode. All AI contributions are attributed in commit messages, PR descriptions, and documentation. The format is:
-
-```text
-feat: implement session TTL auto-renewal
+All commits include AI attribution:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)